images/desktop/ubuntu.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM codercom/enterprise-base:latest
+
+USER root
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
+    apt-get install -y --no-install-recommends --no-install-suggests dbus-x11 openssl ssl-cert libdatetime-perl xfce4 xfce4-goodies && \
+    rm /run/reboot-required* || true
+
+# Setting the required environment variables
+ARG USER=coder
+RUN echo 'LANG=en_US.UTF-8' >> /etc/default/locale; \
+    echo 'export GNOME_SHELL_SESSION_MODE=ubuntu' > /home/$USER/.xsessionrc; \
+    echo 'export XDG_CURRENT_DESKTOP=xfce' >> /home/$USER/.xsessionrc; \
+    echo 'export XDG_SESSION_TYPE=x11' >> /home/$USER/.xsessionrc;
+
+USER coder
```

Comment:
Think this just might be an issue on my end, but running the format checker against the Dockerfile I get this:
```bash
$ dprint check --plugins https://plugins.dprint.dev/dockerfile-0.3.0.wasm https://plugins.dprint.dev/markdown-0.17.0.wasm
from /Users/****/Desktop/Projects/test-templates/enterprise-desktop/ubuntu.Dockerfile:
13 13| ····echo·'export·XDG_CURRENT_DESKTOP=xfce'·>>·/home/$USER/.xsessionrc;·\
14 14| ····echo·'export·XDG_SESSION_TYPE=x11'·>>·/home/$USER/.xsessionrc;
15 15| 
   16|-USER·coder␃
16   |+USER·coder
--
```

Shows an error with there being no newline at the end of `USER coder`. Adding a newline fixes it, but never ran into this when reintroducing the golang, node, and java images. 

If you do see this error though during the build job, this might be the solution just in case.